source/GameInterface/Services/BesiegerCamps/Extensions/BesiegerCampExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using GameInterface.Services.ObjectManager;
+using Serilog;
+using TaleWorlds.CampaignSystem.Siege;
+
+namespace GameInterface.Services.BesiegerCamps.Extensions
+{
+    public static class BesiegerCampExtensions
```

Comment:
This class adds unnecessary complexity. Is there any way to make it work with the current object manager?

Reply:
gone, reduced to atoms